### PR TITLE
[minizip-ng] made cmake installs relative

### PR DESCRIPTION
**Pull request by prateek9623**
**Describe the pull request**

- #### What does your PR fix?  
  After installation minizip cmake installs had some absolute paths which were making them unusable, this PR makes them relative

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Please re-make patch `made_cmake_install_relative.patch`.

**Comment by prateek9623** [comment]
> Please re-make patch `made_cmake_install_relative.patch`.

i tried making, but it was still failing locally

**Comment by JackBoosY** [comment]
I'll handle this.

**Comment by JackBoosY** [comment]
@prateek9623  I can be sure that the all relevant codes in patch `CMakeLists-targets.patch` almost do not exist, please delete `buildtrees` folder and remake this patch.

**Comment by prateek9623** [comment]
I found the issue, it was not with the patch I created, it was just a conflict between patches, as both patches change the same line in code. So, to fix that I need to modify old patch also

**Review by JackBoosY — APPROVED** [review]

**Comment by prateek9623** [comment]
this patch is not required if minizip-ng port is updated to 3.0.2

**Comment by prateek9623** [comment]
@JackBoosY Hey, i have updated minizip-ng version, can you re-review it, thanks

File: versions/m-/minizip-ng.json [hunk 1]
```diff
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "c3f833fb14771ffd8f32496aa12035c86ab1cb9a",
+      "git-tree": "38b52fdd102def83bceebc88722fc88165187695",
+      "version": "3.0.2",
+      "port-version": 0
+    },
+    {
+      "git-tree": "42734467159d3cb26d8d5f095d3da124c9ac2c1b",
```
Comment (JackBoosY) [inline versions/m-/minizip-ng.json:9]:
```suggestion
      "git-tree": "38b52fdd102def83bceebc88722fc88165187695",
      "version": "3.0.2",
      "port-version": 0
    },
    {
      "git-tree": "c3f833fb14771ffd8f32496aa12035c86ab1cb9a",
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Please run command `./vcpkg format-manifest --all` then commit changes.

**Review by JackBoosY — APPROVED** [review]

**Pull request merged**